addons/mail/static/src/components/messaging_menu/messaging_menu.js

```diff
@@ -101,10 +101,10 @@ class MessagingMenu extends Component {
      */
     _useStoreSelector(props) {
```

Comment:
must be done also in overrides

Reply:
Don't see any overrides on these root components that are problematic.

Reply:
Note this method could/should be made static to avoid the issue in the first place, since it's not really using messaging at all.